assets/css/v2/lcd_common_styles/subway_status.scss

```diff
@@ -1,170 +1,172 @@
 .subway-status {
```

Comment:
This file replaced the old `subway_status` styles. It used to be called `subway_status_v2`. No changes were actually made.